examples/nodeconfigs/nosandbox.json

```diff
@@ -0,0 +1,17 @@
+{
+    "kernel_filepath": "/path/to/vmlinux-5.10",
+    "rootfs_filepath": "/path/to/rootfs.ext4",
+    "machine_pool_size": 1,
+    "cni": {
+        "network_name": "fcnet",
+        "interface_name": "veth0"
+    },
+    "machine_template": {
+        "vcpu_count": 1,
+        "memsize_mib": 256
+    },
+    "tags": {
+        "simple": "true"
+    },
+    "no_sandbox": true
```

Comment:
personal preference, double negates confuse me. 
sandbox: [true|false] would be more clear i think

Reply:
I actually used the double negative like this deliberately so people couldn't accidentally disable the sandbox